### ISSUE TITLE
default_icon(browser_action) doesn't exist in resulting manifest.json

### ISSUE THREAD

**Issue by sergey-shpak** [kevincharm/parcel-plugin-web-extension#22]
`default_icon` property doesn't exist in resulting manifest.json when `default_icon` is an object

manifest.json
```json
  "browser_action": {
    "default_icon": {
      "32": "img/icon32.png"
    },
    "default_title": "title",
    "default_popup": "popup.html"
  }
```
expected:
```json
  "browser_action": {
    "default_icon": {
      "32": "img/icon32.png"
    },
    "default_title": "title",
    "default_popup": "popup.html"
  }
```
actual:
```json
  "browser_action": {
    "default_title": "title",
    "default_popup": "popup.html"
  }
```

**Comment by sergey-shpak** [comment]
looks like [processAllIcons](https://github.com/kevincharm/parcel-plugin-web-extension/blob/master/src/ManifestAsset.js#L208) should return processed `icons`

**Comment by kevincharm** [comment]
Fixed in v1.4.1
